### ISSUE TITLE
Typo in the ATDOC algorithm

### ISSUE THREAD

**Issue by bluelg** [KevinMusgrave/pytorch-adapt#10]
Hi, I am the first author of ATDOC, thanks for including our method in such an impressive library. 

There exists a typo in the paper of Eq.(6) (already updated the arxiv version today), where the index k should be replaced with i.
That is to say, in Line 87 of this python file pytorch_adapt/layers/neighborhood_aggregation.py, the correct code would be "logits = (logits ** p) / torch.sum(logits ** p, dim=0)".

Best,

Jian

**Comment by KevinMusgrave** [comment]
Fixed
https://github.com/KevinMusgrave/pytorch-adapt/blob/c0461e10f45858837466e298ce33e0165c07a7c0/src/pytorch_adapt/layers/neighborhood_aggregation.py#L87
